grp/simple.gi

```diff
@@ -1115,8 +1204,10 @@ local nam,e,EFactors,par,expo,prime,result,aut,i;
       e:=EFactors(Gcd(4,par[2]^par[1]-1),expo,2);
     fi;
   elif id.series="2D" then
+    classical:=["O-",2*par[1],par[2]];
     nam:=Concatenation("O",String(par[1]*2),"-(",String(par[2]),")");
-    e:=EFactors(Gcd(4,par[2]^par[1]+1),expo/2,1);
+
+    e:=EFactors(Gcd(4,par[2]^par[1]+1),2*expo,1);
```

Comment:
Why the change from /2 to *2 -- is that part of the bug fix?

Reply:
Yes. That was the underlying bug. However this (slightly cryptic) variant is phased out now in favour of the presentations from Bray/Holt/Roney-Dougal, and currently both are checked against each other to avoid further implementation errors.